practice-one/src/styles/layouts/header.css

```diff
@@ -0,0 +1,38 @@
+.main-header {
+  display: flex;
+  justify-content: space-between;
+  padding: 80px 80px;
+
+  & .header-logo {
```

Comment:
If it's not necessary, there's no need to nest the class. Please help to check the other places